bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1222,7 +1222,7 @@ spec:
                   value: /opt/manifests
                 - name: ODH_PLATFORM_TYPE
                   value: OpenDataHub
-                image: REPLACE_IMAGE:latest
+                image: quay.io/opendatahub/opendatahub-operator:latest
```

Comment:
this probably should be reverted

Reply:
Will do, what causes this change is it `make generate manifests api-docs bundle`

Reply:
yep, we may need to do something better to avoid updating this file by mistake